### ISSUE TITLE
Sorting on collection attribute

### ISSUE THREAD

**Issue by gmjordan** [ArangoDB-Community/arangodb-graphql-spring-boot-starter#4]
Is sorting provided for on edge and vertex collections?

**Comment by colinfindlay** [comment]
Hi Greg, sorry for the delayed response - I've been on vacation.

There is currently no support for sorting. The AQL query that is generated basically outputs a list of paths which is probably not particularly useful for sorting, other than perhaps the top level root vertices.

I assume by the question you are looking to sort nested relationships so something like the following pseudo-GraphQL query 

`
{
  clients {
    name @sort
    age
    address {
        line1
        line2
        city @sort
   }
}
`

i.e. Sort Clients by name, sort each client's address by city.

 Is that right?

**Comment by gmjordan** [comment]
yes, but top level sort would cover many use cases. For example, list of accounts ordered by account # or "friendly" name on the account, e.g. checking, savings ArangoDB-Community/arangodb-graphql-spring-boot-starter#1 , savings ArangoDB-Community/arangodb-graphql-spring-boot-starter#2  

it would be ideal to have this with skip and limit, also, for pagination
